EventsExpress/ClientApp/src/components/Route guard/401.js

```diff
@@ -60,7 +59,6 @@ const mapDispatchToProps = (dispatch) => {
            dispatch(getUsersError(false));
            dispatch(getEventError(false));
            dispatch(setAvatarError(false));
-           dispatch(set2CommentError(null));
            dispatch(set1CommentError(false));
```

Comment:
Do we still need 1 and 3? :) Maybe remove the `error` from redux for comments object altogether?